libV2/utils/func.js

```diff
@@ -0,0 +1,20 @@
+const { callbackify } = require('util');
+
+/**
+ *  Convenience function to handle "if no callback then return a promise" pattern
+ *
+ * @param {Function} asyncFunc - asyncFunction to call
+ * @param {Function|undefined} callback - optional callback
+ * @returns {Promise|undefined} - returns a Promise if no callback is passed
+ */
+function asyncOrCallback(asyncFunc, callback) {
```

Comment:
Should this util be in arsenal?

Reply:
Is there a need in other projects?

Reply:
could be and this could be a chore/improvement later.